apps/dsp-app/src/app/project/ontology/ontology-form/ontology-form.component.html

```diff
@@ -1,70 +1,39 @@
-<form [formGroup]="ontologyForm" (ngSubmit)="submitData()" class="form">
-  <div class="form-content">
-    <h3 *ngIf="!iri">Create new ontology</h3>
-    <mat-form-field class="large-field ontology-name">
-      <input
-        matInput
-        class="ontology-name"
-        autocomplete="off"
-        data-cy="name-input"
-        [maxlength]="nameMaxLength"
-        [formControl]="ontologyForm.controls['name']"
-        [placeholder]="'Set a unique name'" />
-      <mat-icon
-        matSuffix
-        [matTooltip]="ontologyNameInfo"
-        matTooltipPosition="above"
-        matTooltipClass="multi-line-tooltip"
-        >help</mat-icon
-      >
-      <mat-hint *ngIf="formErrors.name" class="medium-field"> {{ formErrors.name }} </mat-hint>
-      <mat-hint align="end">{{ ontologyForm.controls['name'].value.length }} / {{ nameMaxLength }}</mat-hint>
-    </mat-form-field>
-
-    <mat-form-field class="large-field ontology-label">
-      <span *ngIf="!iri" matPrefix>{{ project.shortname }}:&nbsp;</span>
-      <input
-        matInput
-        data-cy="label-input"
-        placeholder="Label"
-        [formControl]="ontologyForm.controls['label']"
-        [value]="ontologyLabel" />
-      <mat-hint *ngIf="formErrors.label"> {{ formErrors.label }} </mat-hint>
-    </mat-form-field>
-
-    <mat-form-field class="large-field ontology-comment">
+<app-dialog-header
+  [title]="ontologyName ? ontologyName : 'Create new ontology'"
+  [subtitle]="ontologyName ? 'Customize Data model' : ''">
```

Comment:
Make subtitle consistent in create and edit dialogs. There are two different concepts - ontology and data model. 
Also use translation labels

Reply:
Sadly, this is how it is consistent with other cases and this is exactly as it has been. I don't like it at all, but when I refactor, I keep the design and what is displayed as text the very same. I streamline eventually like using the new side dialog.

Reply:
I changed it to "data model" as it seems to be in use more often. I'll make a note for RDU.

Reply:
Translations are a task I would prefer to do in a separate PR for several/all translations at once. There are imho other things to clean up concerning the translations, e.g. the structure, remove unused, duplicates ...

Reply:
translations are implemented now - however I dislike mixing new features and refactoring in a refactor PR. And I still feel that the translations should be checked and changed separately before adding new translations.

Reply:
Let's update translations in the next cycle, though it will be quite a big task